.github/workflows/client.yml

```diff
@@ -28,15 +29,47 @@ jobs:
       - run: npm run lint --workspace packages/client
       - run: npm run build --workspace packages/client
       - run: npm test --workspace packages/client
-  typedoc:
+
+  release:
+    name: Release
     runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v3
+        id: tag-release
+        with:
+          path: packages/client
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          monorepo-tags: true
+          package-name: w3name
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Other Changes","hidden":false}]'
+      - uses: actions/checkout@v2
+        if: ${{ steps.tag-release.outputs.releases_created }}
+      - uses: actions/setup-node@v2
+        if: ${{ steps.tag-release.outputs.releases_created }}
+        with:
+          node-version: '16'
+          registry-url: https://registry.npmjs.org/
+      - uses: bahmutov/npm-install@v1
+        if: ${{ steps.tag-release.outputs.releases_created }}
+      # --- Client deploy steps  ---------------------------------------------
+      - name: Client - NPM Publish
+        if: ${{ steps.tag-release.outputs.releases_created }}
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
```

Comment:
Where is `secrets.NPM_AUTH_TOKEN` set? Do we need to give the client package access to whatever account this token repreesents?

Reply:
Good point, I've asked someone to set this up for us.

Reply:
Looks like it is now set with the bot added to the package! Let's give this a test run